integration-tests/contract-permits.spec.ts

```diff
@@ -247,13 +247,13 @@ CONFIGS().forEach(({ lib, rpc, setup, createAddress }) => {
       done();
     });
 
-    describe(`Test of contracts having a permit for tzip-17: ${rpc}`, () => {
+    describe(`Test contract having a permit for tzip-17: ${rpc}`, () => {
       beforeEach(async (done) => {
         await setup(true);
         done();
       });
 
-      test('Show that any user can submit the permit hash to use an entrypoint', async (done) => {
+      test('As a User I want to verify that I can submit the permit hash to use an entrypoint\n', async (done) => {
```

Comment:
As a User I want to verify -> Verify

~\n~

Reply:
rewritten